libtenzir/include/tenzir/node.hpp

```diff
@@ -34,39 +34,39 @@ struct node_state {
     -> const handler_and_endpoint&;
 
   /// The REST endpoint handlers for this node. Spawned on demand.
-  std::unordered_map<std::string, handler_and_endpoint> rest_handlers = {};
+  std::unordered_map<std::string, handler_and_endpoint> rest_handlers;
```

Comment:
While I personally find this new style more pleasing to look at, we use the `= {}` form extensively all over the codebase. I think we should rather remove that clang-tidy warning and keep it consistent.

Reply:
The warning does have a distinct advantage: It also works the other way around, telling you to initialize trivial types that otherwise would contain garbage values. I think it's better to leave it enabled because of that.